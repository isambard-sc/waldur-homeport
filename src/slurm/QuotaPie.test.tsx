import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { QuotaPie } from './QuotaPie';

describe('Quota pie', () => {
  it('renders SVG chart', () => {
    expect(render(<QuotaPie value={0.33} />).container).toMatchSnapshot();
  });
});
