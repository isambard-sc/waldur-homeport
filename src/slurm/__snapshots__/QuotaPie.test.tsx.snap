// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Quota pie > renders SVG chart 1`] = `
<div>
  <svg
    height="15px"
    rotate="-90deg"
    viewBox="-1 -1 2.1 2.1"
  >
    <path
      d="M 1 0 A 1 1 0 0 1 -0.48175367410171543 0.8763066800438635 L 0 0"
      fill="Coral"
    />
    <path
      d="M -0.48175367410171543 0.8763066800438635 A 1 1 0 1 1 1 -2.4492935982947064e-16 L 0 0"
      fill="#00ab6b"
    />
  </svg>
</div>
`;
